recipes-devtools/cargo/files/0001-disable-cargo-snapshot-fetch.patch

```diff
@@ -8,20 +8,20 @@ Subject: [PATCH] disable cargo snapshot fetch
  1 file changed, 4 deletions(-)
 
 diff --git a/Makefile.in b/Makefile.in
-index 286a593..9c66486 100644
+index fadcb4b..a8da630 100644
 --- a/Makefile.in
 +++ b/Makefile.in
-@@ -92,10 +92,6 @@ test-unit-$(1): $$(CARGO)
+@@ -103,10 +103,6 @@ test-unit-$(1): $$(CARGO)
  endef
  $(foreach target,$(CFG_TARGET),$(eval $(call CARGO_TARGET,$(target))))
  
--$(TARGET_ROOT)/snapshot/bin/cargo$(X): src/snapshots.txt
--	$(CFG_PYTHON) src/etc/dl-snapshot.py $(CFG_BUILD)
+-$(TARGET_ROOT)/snapshot/bin/cargo$(X): $(S)src/snapshots.txt
+-	$(CFG_PYTHON) $(S)src/etc/dl-snapshot.py $(CFG_BUILD)
 -	touch $@
```

Comment:
The removal isn't strictly required, but it does make sure that we actually grab the snapshot ourselves instead of having cargo's build download it behind our backs (which has happened a few times in the past when our updates weren't quite right)